Makefile

```diff
@@ -20,197 +20,47 @@
 
 SRC?=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
-### Beginning of user-adjustable variables ###
+include $(SRC)/Makefile.common
```

Comment:
I wonder, is it possible to conditionally include these files based on the target? It will stop me having to specify a Linux kernel dir for a Windows image, which is a bit weird. Maybe this isn't possible, I'm no Make guru haha :)